### DISCUSSION TITLE
Raspberry 7 inch touch display 2 not working out of the box

### DISCUSSION THREAD

**Discussion by olialb** [guysoft/FullPageOS#609] [Show and tell]
I am using a raspberry pi 4 and the new official touch display 2 with the latest nightly build of FullPageOS
After setting it up as described here the display shows the full page web site which I configured but touch is not working.
Even after updating the system with 

sudo apt update
sudo apt upgrade

still no touch functionality.

I analyzed the difference between config.txt in the offcial 32Bit OS and FullPageOS and find the line:

start_x=1

This option is set in FullPageOS. It seams to be used to enable the camera!? 
Anyway.... After I deleted this line in config.txt touch is now working fine!!

Just as an info for other users.
Thanks for this cool project!

**Comment by guysoft** [comment]
Not sure from the docs what its does neither:
https://www.raspberrypi.com/documentation/computers/legacy_config_txt.html#start_x-start_debug

**Comment by guysoft** [comment]
Looks like some kind of firmware dark magic done on RaspberryPi side that is undocumented, it loads some code probably that is used both in the camera and display:
https://github.com/raspberrypi/firmware/issues/959#issuecomment-376182663

We could add this to the config.txt or the FAQ if it breaks something else.

**Reply by apartmedia** [reply]
> Looks like some kind of firmware dark magic done on RaspberryPi side that is undocumented, it loads some code probably that is used both in the camera and display: [raspberrypi/firmware#959 (comment)](https://github.com/raspberrypi/firmware/issues/959#issuecomment-376182663)
> 
> We could add this to the config.txt or the FAQ if it breaks something else.

Also strange that your linked comment says enabling this setting loads „full firmware“, suggesting it will support „all“ features. Makes it even more crazy that it disables touch function on the official touch display 🤔

**Comment by olialb** [comment]
Yes, the functinality of start_x=1 seams to be not fully documented. 
I just can say that its reproducable: With this option set touch for touch display 2 is not working.
But for sure I can not say if it breaks something else...

I am just happy now that I have FullPageOS running with touch display 2 and Raspi 4 :-)
